### PR TITLE
Add iOS AppStore links for GoMap!! and EveryDoor; Update URL for GoMap!!

### PR DESCRIPTION
**Pull request by habi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
...
 * [Vespucci](https://vespucci.io/) - Advanced full-data-model editor for Android. ([Android](https://play.google.com/store/apps/details?id=de.blau.android) / [Source Code](https://github.com/MarcusWolschon/osmeditor4android) / [Wiki](https://wiki.openstreetmap.org/wiki/Vespucci))
-* [GoMap!!](https://apps.apple.com/app/id592990211) - Feature-rich mobile editor for iOS. (iOS / [Source Code](https://github.com/bryceco/GoMap) / [Wiki](https://wiki.openstreetmap.org/wiki/Go_Map!!))
-* [Every Door](https://every-door.app) - Every Door is an Android and iOS app that lets you create and edit objects in OpenStreetMap. It supports editing node and polygon features, and creating point features like shops, benches, and building entrances. ([Android](https://play.google.com/store/apps/details?id=info.zverev.ilya.every_door) / [Source Code](https://github.com/zverik/every_door) / [Wiki](https://wiki.openstreetmap.org/wiki/Every_Door))
+* [GoMap!!](https://apps.apple.com/app/id592990211) - Feature-rich mobile editor for iOS. ([iOS](https://apps.apple.com/app/id592990211) / [Source Code](https://github.com/bryceco/GoMap) / [Wiki](https://wiki.openstreetmap.org/wiki/Go_Map!!))
```
Comment (Unkn0wnCat) [inline README.md:70]: The iOS App is already linked in the project name at the start of the line, and using the same link twice is not up to code, sorry 🙁 

```suggestion
* [GoMap!!](https://apps.apple.com/app/id592990211) - Feature-rich mobile editor for iOS. (iOS / [Source Code](https://github.com/bryceco/GoMap) / [Wiki](https://wiki.openstreetmap.org/wiki/Go_Map!!))
```

File: README.md [hunk 2]
```diff
@@ -67,8 +67,8 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
...
+* [Every Door](https://every-door.app) - Every Door is an Android and iOS app that lets you create and edit objects in OpenStreetMap. It supports editing node and polygon features, and creating point features like shops, benches, and building entrances. ([Android](https://play.google.com/store/apps/details?id=info.zverev.ilya.every_door) / [iOS](https://apps.apple.com/app/id1621945342) / [Source Code](https://github.com/zverik/every_door) / [Wiki](https://wiki.openstreetmap.org/wiki/Every_Door))
```
Comment (Unkn0wnCat) [inline README.md:71]: This is a great addition, thanks!

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
The change on `Every Door` is great, but the update on `GoMap!!` introduces a duplicate link

File: README.md (hunk 2 above)
Comment (habi) [reply README.md:71]: Very welcome!

File: README.md (hunk 1 above)
Comment (habi) [reply README.md]: I've now updated my PR with the 'correct' link to http://gomaposm.com, which is the (very minimal) website of Go Map!!

**Review by Unkn0wnCat — APPROVED** [review]
Looking good 👍🏼

**Comment by Unkn0wnCat** [comment]
Thanks, that is certainly *a* website 😛 - thanks for the contribution, as I agree the link was somewhat weird with the `iOS` not being linked anywhere! 🥇

**Pull request merged**